i18n/fr.toml

```diff
@@ -69,25 +72,31 @@ other = "Chercher"
 other = "Rechercher des titres, des contenus..."
 
 [clear]
-other = "Clair"
+other = "Effacer"
 
 [cancel]
 other = "Annuler"
 
 [noResultsFound]
 other = "Aucun résultat trouvé"
 
+[lunrLanguageCode]
+other = "fr"
+
+[lunrLanguageLib]
```

Comment:
is it useless ?

Reply:
Yes, Lunr is deprecated in favour of [Fuse.js](https://fusejs.io/).

Reply:
Ok, so i remove it